exercises/alphametics/description.md

```diff
@@ -1,6 +1,6 @@
 # Description
 
-Write a function to solve alphametics puzzles.
+Write a solver for alphametics puzzles.
```

Comment:
What is a "solver"? Should this just ask the student to solve the puzzle? Write code to solve the puzzle?